backend/src/routes/user.routes.ts

```diff
@@ -1,12 +1,29 @@
 import express from 'express';
+import { DatabaseFactory } from '../database/database-factory';
+import { BaseUser } from '../models/user';
 
 /**
  * A router for all paths beginning with /users
  */
 const userRouter = express.Router({ mergeParams: true });
 
-userRouter.post('/', (req, res) => {
-  res.send('create user');
+userRouter.post('/', async (req, res) => {
+  const body = req.body;
+  const { email, first_name, last_name, passwd } = body;
+  const newUser: BaseUser = {
+    email,
+    first_name,
+    last_name,
+    passwd,
+    unallocated_funds: 0,
+  };
+  const db = DatabaseFactory.getDatabase();
+  try {
+    const user = await db.addUser(newUser);
+    res.send(user);
+  } catch (error) {
+    res.send(error.message);
+  }
```

Comment:
Create user route; example of how to actually use the database, the requests, and the response.